awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -195,8 +198,32 @@ class JobOutput extends Component {
     }
   }
 
+  loadTracebackEvents() {
+    // When a job has traceback results, render it as a job event
+    // with the traceback as the stdout text
+    const { job } = this.props;
+    const remoteRowCount = 1;
+    const results = [
+      {
+        counter: -1,
+        created: null,
+        event: null,
+        type: null,
+        stdout: job?.result_traceback || '',
```

Comment:
is there a case where there would be a traceback that does not have a value in `job.result_traceback`?  If so, rather than show an empty string should we show some default string value instead?

Reply:
In most cases, if status is "error" then the result_traceback should be non-blank. However, I would prefer not to rely on that assumption. It's also possible (although maybe not important) that the normal standard out can be non-blank and the result_traceback can be non-blank due to inventory updates hitting a license error after attempting to save hosts (DB transaction gets rolled back).

I don't want to be opinionated, just trying to help define the envelope of possible API behaviors.

Reply:
@AlexSCorey 

That's a good question. 

No - At this point, this code path isn't reached _unless_ `result_traceback` is defined: https://github.com/jakemcdermott/awx/blob/fix-8709/awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx#L221

The `''` default I'm using for this is just some defensive coding on my part in case that changes in the future.

Reply:
> In most cases, if status is "error" then the result_traceback should be non-blank. However, I would prefer not to rely on that assumption. 

This is some good info, thanks @AlanCoding. The way I've currently implemented this doesn't rely on this assumption, so I think we are good here. The actual value of `status` isn't considered at all. We only check `results_traceback` when deciding to render the traceback instead of stdout.

> It's also possible (although maybe not important) that the normal standard out can be non-blank and the result_traceback can be non-blank due to inventory updates hitting a license error after attempting to save hosts (DB transaction gets rolled back).

@AlanCoding 

In this scenario, the normal standard out would be ignored and we'd only show the traceback. 

It sounds like showing _both_ might be useful in this scenario, though. Do you have a test case I can use to reproduce the scenario locally? I'll see about squeezing in some changes to handle both outputs.

Reply:
I think violating the organization `max_hosts` is the best way to hit this.